matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/LoadTimelineStrategy.kt

```diff
@@ -309,8 +310,8 @@ internal class LoadTimelineStrategy constructor(
     /**
      * Clear any existing thread chunk.
      */
-    private fun clearThreadChunkEntity(realm: Realm, rootThreadEventId: String) {
-        realm.executeTransaction {
+    private suspend fun clearThreadChunkEntity(realm: Realm, rootThreadEventId: String) {
+        awaitTransaction(realm.configuration) {
```

Comment:
I am wondering if it would be safer to use `realm.executeTransactionAwait` which has the advantage to perform a `refresh()` at the end of the transaction, rather than our home made `awaitTransaction`.